install/openshift/roles.yaml

```diff
@@ -2,6 +2,23 @@ apiVersion: v1
 kind: List
 items:
 - apiVersion: v1
+  kind: ClusterRole
```

Comment:
this cluster role is a default part of openshift now so we don't need it anymore.